docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -69,6 +69,14 @@ You can also use Task Manager in {kib} to troubleshoot background tasks and proc
 * {kibana-ref}/task-manager-health-monitoring.html[Task Manager health monitoring]
 * {kibana-ref}/task-manager-troubleshooting.html[Task Manager troubleshooting]
 
+[float]
+[[troubleshoot-max-alerts]]
+==== Troubleshoot max alerts warning
+
+When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following warning is displayed on the rule's details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` 
```

Comment:
```suggestion
When a rule reaches the maximum number of alerts it can generate during a single rule execution, the following warning appears on the rule's details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` 
```